Stab at adding stderr differentiation

seanpue opened the pull request:
This attempts to add a separate output style for `stderr`, addressing #71. 

It adds a `literal_block` without highlighting (by not settings `rawsource` in `literal_block()`, which disables Pygment), inside a `container`. Both are set to classes `error` and `stderr`. `Error` is common in Sphinx themes, especially as a `div` (container) style, whereas `stderr`, applied after, is not, and will allow CSS customization by the user for this particular output.

The HTML result is the following:
```
<div class="error stderr docutils container">
<pre class="error stderr literal-block">{{ ERROR MESSAGE}}
</pre>
</div>
```

LatexPDF results are differentiated by the absence of borders. Tested in Alabaster and sphinx_rtd_theme.

jbweston commented:
Nice one! Can you post a screenshot of what this looks like with the standard sphinx theme?

seanpue commented:
Here it is without any CSS adjustments in Alabaster. (Alabaster has a background color defined for `pre`, so the text background is showing up gray).
<img width="676" alt="Screen Shot 2019-10-08 at 3 06 52 PM" src="https://user-images.githubusercontent.com/6502057/66425935-c34b1500-e9de-11e9-96e6-d2700133b559.png">

Here it is with a custom.css changing background color for `stderr`:
```
.stderr {
    background-color: #FCC
}
```
<img width="674" alt="Screen Shot 2019-10-08 at 3 20 31 PM" src="https://user-images.githubusercontent.com/6502057/66426149-39e81280-e9df-11e9-8d57-fb0dca6adb81.png">

And here is the PDF output:

<img width="760" alt="Screen Shot 2019-10-08 at 3 22 32 PM" src="https://user-images.githubusercontent.com/6502057/66426287-79166380-e9df-11e9-850f-8ee82df87af0.png">

akhmerov commented:
Nice!

I think the only thing remaining to do here is updating the docs to remove the warning about the lack of styling.

seanpue commented:
Great! I removed the warning, added a note with a hyperlink hint about how to adjust the CSS styling in the default theme, added a `custom.css` to fix the background color in `doc/source/_static`, and adjusted `conf.py` to load it.

jbweston commented:
Closes #71
